specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -251,6 +257,51 @@ function Merging({
   );
 }
 
+function MergeButton<SCHEMA extends AnySchema>({
+  id,
+  mergeResource,
+}: {
+  readonly id: (suffix: string) => string;
+  readonly mergeResource: SpecifyResource<SCHEMA>;
+}): JSX.Element {
+  const [formId] = React.useState(id('form'));
+  const [saveBlocked, setSaveBlocked] = React.useState(false);
+  const [showSaveBlockedDialog, setShowBlockedDialog] = React.useState(false);
+
+  React.useEffect(() => {
+    setSaveBlocked(false);
+    return resourceOn(
+      mergeResource,
+      'blockersChanged',
+      (): void => {
+        const onlyDeferredBlockers = Array.from(
+          mergeResource.saveBlockers?.blockingResources ?? []
+        ).every((resource) => resource.saveBlockers?.hasOnlyDeferredBlockers());
+        setSaveBlocked(!onlyDeferredBlockers);
+      },
+      true
+    );
+  }, [mergeResource]);
+
+  return (
+    <>
+      {!saveBlocked ? (
+        <Submit.Blue form={formId}>{treeText.merge()}</Submit.Blue>
+      ) : (
+        <Submit.Red className="cursor-not-allowed">
```

Comment:
while you make the button look not clickable, does it actually prevent the button click from submitting the form?

also, try disabling the button to see if you like the styling of the disabled button (might remove the need for adding `cursor-not-allowed` manually)

Reply:
> while you make the button look not clickable, does it actually prevent the button click from submitting the form?
Yes, this always prevents the button click from submitting the form. I don't pass in the formId into the form prop for this purpose. 

When the formId is passed in as a form prop, there _is_ a bug where the user can submit the form where there are errors (saveblockers or validation errors) in any subview. 
I now know to take a look at the browser API and ensure the parent resource is invalid when a Merging Subview is invalid